velox/expression/fuzzer/SparkExpressionFuzzerTest.cpp

```diff
@@ -65,8 +65,9 @@ int main(int argc, char** argv) {
       "replace",
       "might_contain",
       "unix_timestamp",
-      // from_unixtime throws VeloxRuntimeError when the timestamp is out of the
-      // supported range.
+      // date_format and from_unixtime throws VeloxRuntimeError when the
+      // timestamp is out of the supported range.
+      "date_format",
```

Comment:
The `VeloxRuntimeError` with `UNSUPPORTED_INPUT_UNCATCHABLE` code is allowed in the fuzzer test. I wonder if we can throw it in the 'from_unixtime' and 'date_format' to make the fuzzer test work. Seeing details in https://github.com/facebookincubator/velox/commit/8a6ab15696bb1ca68a4d69678e855e00f5bc8742.